lib/galaxy/datatypes/converters/parquet_to_csv_converter.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+"""
+Input: parquet
+Output: csv
+"""
+import os
+import sys
+try:
+    import pyarrow
+except ImportError:
+    pyarrow = None
+
+
+def __main__():
+    infile = sys.argv[1]
+    outfile = sys.argv[2]
+
+    if not os.path.isfile(infile):
+        sys.stderr.write(f"Input file {infile!r} not found\n")
+        sys.exit(1)
+
+    if pyarrow is None:
+        raise Exception("Cannot run conversion, pyarrow is not installed.")
+    pyarrow.csv.write_csv(pyarrow.parquet.read_table(infile), outfile)
```

Comment:
That doesn't seem to work:
```
galaxy.tool_util.verify.interactor.JobOutputsError: Job in error state.. tool_id: CONVERTER_parquet_to_csv, exit_code: 1, stderr: Traceback (most recent call last):
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/datatypes/converters/parquet_to_csv_converter.py", line 28, in <module>
    __main__()
  File "/home/runner/work/galaxy/galaxy/galaxy root/lib/galaxy/datatypes/converters/parquet_to_csv_converter.py", line 24, in __main__
    pyarrow.csv.write_csv(pyarrow.parquet.read_table(infile), outfile)
  File "/home/runner/miniconda3/envs/__pyarrow@4.0.1/lib/python3.9/site-packages/pyarrow/__init__.py", line 259, in __getattr__
    raise AttributeError(
AttributeError: module 'pyarrow' has no attribute 'csv'
.
```

Reply:
you may have to `import pyarrow.csv`

Reply:
probably resolved with https://github.com/galaxyproject/galaxy/pull/13078/commits/fad9eeab89c29457bc0b39494a2f578a6d5a9e9e